ui/ducks/app/app.js

```diff
@@ -388,6 +405,19 @@ export function hideWhatsNewPopup() {
 export function toggleGasLoadingAnimation(value) {
   return { type: actionConstants.TOGGLE_GAS_LOADING_ANIMATION, value };
 }
+export function addTransactionToDisplayOnFailure(txId) {
+  return {
+    type: actionConstants.ADD_TRANSACTION_TO_DISPLAY_ON_FAILURE,
+    value: txId,
+  };
+}
+
+export function removeTxFromFailedTxesToDisplay(txId) {
```

Comment:
Sorry I had meant that both functions should be renamed. This function should be renamed too, for the same reason as `addTransactionToDisplayOnFailure`.

Reply:
I'm very bad with names...so this is what I come up with

`removeTransactionsDisplayedOnFailure` and `REMOVE_TRANSACTIONS_DISPLAYED_ON_FAILURE`

Reply:
Hmm.... to be the obvious choice would be to match the parallel "add" function, but replace "add" with "remove". e.g.

`REMOVE_TRANSACTION_TO_DISPLAY_ON_FAILURE` and `removeTransactionToDisplayOnFailure`